### PR TITLE
Update UG with comments by Serene and Hanming

### LINKED ISSUE #129: Update UG with comments from Serene
Feedback for UG by Serene

Introduction
- Bold words
- Branding of GradPad
- Who are we
- Be more boastful in the efficiency of the app
- Something like a written demo (How syafiq can use this app)

QuickStart
- Define GUI/change to mainpage
- Brief intro/overview of the section (What we will be going through in this section)

- Display rationale of find/delete feature first before explaining feature

- Checkmc - explanation of modular credits

- Personalize GradPad (Name, Academic year etc...)

- Better navigation (Numbered sections)
- Varied Font sizes
- Compare it to Apple UserGuide
- Numbered labels to diagram with description of labels below

- Explain why help feature is placed at the start

### PR DESCRIPTION
**Pull request by shaokiat**
Feedback addressed in UG:

Introduction
- Bold words
- Branding of GradPad
- Who are we

<br/>

- Checkmc - explanation of modular credits
- Brief overview of Feature Summary List, Command List

Comments by Hanming:
- Place more emphasis on effort made by team so it is hard to miss
- Include a section on the use of NUSmods integration in GradPad
- Emphasize use of NUSmods in add, search, edit command

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -10,19 +10,23 @@ title: User Guide
 --------------------------------------------------------------------------------------------------------------------
 ## GradPad
 
-GradPad is a one-stop solution to module management for Computer Science Undergraduates.
+We are a team of **Computer Science Undergraduates** working together to solve **module management** and **graduation planning** problems
+faced by fellow Computer Science students. Below is the detailed **User Guide** for effective use of GradPad which can help you
+ease and improve your **modules planning process**.
```
Comment (Silvernitro) [inline docs/UserGuide.md]: Grammar nit: 
```suggestion
ease and improve your **module planning process**.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -10,19 +10,23 @@ title: User Guide
...
+
+GradPad is a one-stop solution to **module management** for Computer Science Undergraduates.
 Planning for modules has always been a tedious process but it does not have to be.
 
 The current approach to planning and tracking graduation requirements is to open up tabs after tabs of NUS resources which can
```
Comment (Silvernitro) [inline docs/UserGuide.md]: I think Serene mentioned that we can list examples of the "NUS resources" we are talking about. Maybe we can say NUSMods and/or faculty websites?

File: docs/UserGuide.md (hunk 1 above)
Comment (shaokiat) [reply docs/UserGuide.md]: Edited! Thanks for the catch

File: docs/UserGuide.md (hunk 2 above)
Comment (shaokiat) [reply docs/UserGuide.md]: Yup I've added them in. Should I include hyper links to NUSMods and website of faculty of computing?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -192,7 +199,11 @@ To delete the module:
 
 ### Checking total modular credits: `checkmc`
```
Comment (yan-soon) [inline docs/UserGuide.md:262]: Maybe we can add the Scenario of when we might want to use the command?

File: docs/UserGuide.md (hunk 2 above)
Comment (Silvernitro) [reply docs/UserGuide.md]: Hmm, I didn't think of that but since you've suggested it, why not? Haha I think there's no harm and it might be good :D

**Review by Silvernitro — APPROVED** [review]

**Review by yan-soon — APPROVED** [review]

File: docs/UserGuide.md [hunk 4]
```diff
@@ -10,21 +10,39 @@ title: User Guide
...
+--------------------------------------------------------------------------------------------------------------------
+## NUSMods integration  
+
+![NUSMods](images/nusmods.png)
+
+To streamline your module searching process with added convenience, GradPad integrates data from [NUSMods](https://nusmods.com/). With this, 
+you no longer have to visit NUSMods separately to look up module information. Instead, GradPad offers you all that data 
+in the same app you use to plan your modules itself.
+
+We have requested and was granted permission from NUSMods to utilise their platform to retrieve data such as NUS course catalogue
```
Comment (yan-soon) [inline docs/UserGuide.md]: Grammar nit: NUS course catalogues

File: docs/UserGuide.md [hunk 5]
```diff
@@ -10,21 +10,39 @@ title: User Guide
...
+and **module information**. These data are retrieved on the go in **realtime**, using the data provided by NUSMods.
```
Comment (yan-soon) [inline docs/UserGuide.md]: Sentence structure nit: These data are retrieved on the go in **realtime**, directly from NUSMods.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -37,7 +55,10 @@ Follow the steps below to learn how to get started with GradPad!
 
 1. Copy the file to the folder you want to use as the _home folder_ for your GradPad.
 
-1. Double-click the file to start the app. The Main Page similar to below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Ensure you have **internet connectivity** to fully utilise GradPad with the latest data from [NUSMods](https://nusmods.com/).
+
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
```
Comment (yan-soon) [inline docs/UserGuide.md]: Think we can replace GUI with Main Page?

File: docs/UserGuide.md (hunk 4 above)
Comment (yan-soon) [reply docs/UserGuide.md]: I think if we bold module info we can bold NUS course catalogues as well

**Pull request merged**